src/Controller/AppController.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace CreatorModifier\Controller;
+
+use App\Controller\AppController as BaseController;
+
+class AppController extends BaseController {
+}
```

Comment:
Same question: Is having an AppController in a plugin mandatory? Can this be removed?